server/api/admin/index.ts

```diff
@@ -35,6 +65,10 @@ router.delete("/image/delete", async (req: Request, res: Response) => {
 });
 
 router.get("/images", (_req: Request, res: Response) => {
+  if (!fs.existsSync("server/data/_hidden/image-list.json")) {
```

Comment:
maybe instead of doing this, we should have a file prebuilder alongside the directory prebuilder

Reply:
good point - i'll put it into a PR this week, since it's a more widespread issue